projects/ui/src/shim.cds-core.scss

```diff
@@ -1143,6 +1151,11 @@
       border: 1px solid var(--clr-alert-danger-border-color);
     }
   }
+
+  // TODO(CDE-884): Clean up accordion styles and properly separate accordion/stepper styles
```

Comment:
Added this instead of adding 2 variables due to the media query. I filed a followup ticket to address the underlying styling issues.